### DISCUSSION TITLE
In case it's useful, some more reg keys

### DISCUSSION THREAD

**Discussion by Darthagnon** [TheBobPony/MSEdgeTweaker#5] [Ideas]
I had a similar idea to you and made a script at work a few months ago to make the default browser usable. Just in case it's useful, here's my `Edgium Config.reg`

```
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge]
"HubsSidebarEnabled"=dword:00000000
"AllowSurfGame"=dword:00000000
"AutofillCreditCardEnabled"=dword:00000000
"SmartScreenEnabled"=dword:00000000
"EdgeShoppingAssistantEnabled"=dword:00000000
"ShowMicrosoftRewards"=dword:00000000
"DownloadRestrictions"=dword:00000000
"PasswordMonitorAllowed"=dword:00000000
"AdsSettingForIntrusiveAdsSites"=dword:00000002
"EnterpriseModeSiteListManagerAllowed"=dword:00000001
"InternetExplorerIntegrationTestingAllowed"=dword:00000001
"AutoplayAllowed"=dword:00000000
"PersonalizationReportingEnabled"=dword:00000000
"UserFeedbackAllowed"=dword:00000000
"FamilySafetySettingsEnabled"=dword:00000000
"PaymentMethodQueryEnabled"=dword:00000000
"BingAdsSuppression"=dword:00000000
"ConfigureFriendlyURLFormat"=dword:00000001
"ComponentUpdatesEnabled"=dword:00000001
"PromotionalTabsEnabled"=dword:00000000
"AddressBarMicrosoftSearchInBingProviderEnabled"=dword:00000000
"WebWidgetAllowed"=dword:00000000
"WebCaptureEnabled"=dword:00000000
"NewTabPageHideDefaultTopSites"=dword:00000001
"NewTabPageContentEnabled"=dword:00000000
"NewTabPageAppLauncherEnabled"=dword:00000000
"AllowGamesMenu"=dword:00000000
"VisualSearchEnabled"=dword:00000000
"QuickSearchShowMiniMenu"=dword:00000000
"WebSelectEnabled"=dword:00000000
"WebWidgetIsEnabledOnStartup"=dword:00000000
"ShowAcrobatSubscriptionButton"=dword:00000000
"TyposquattingCheckerEnabled"=dword:00000000
"SiteSafetyServicesEnabled"=dword:00000000
"DiagnosticData"=dword:00000000
"EdgeDiscoverEnabled"=dword:00000000
"AlternateErrorPagesEnabled"=dword:00000000
"SpotlightExperiencesAndRecommendationsEnabled"=dword:00000000
"PasswordGeneratorEnabled"=dword:00000000
"AskBeforeCloseEnabled"=dword:00000000
"HomepageIsNewTabPage"=dword:00000000
"AllowWebContentOnNewTabPage"=dword:00000001
"StartupBoostEnabled"=dword:00000001
"CommandLineFlagSecurityWarningsEnabled"=dword:00000000
"DefaultSearchProviderContextMenuAccessAllowed"=dword:00000000
"AddressBarMicrosoftSearchInBingProviderEnabled"=dword:00000000
"DefaultNotificationsSetting"=dword:00000002

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge\ExtensionInstallAllowlist]

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge\ExtensionInstallBlocklist]
"oejmdnfdoeolakomanjkbmkeheflkmpp"=""

[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge\Recommended]
"BackgroundModeEnabled"=dword:00000000
"SmartScreenForTrustedDownloadsEnabled"=dword:00000000
"SmartScreenPuaEnabled"=dword:00000000
"DefaultDownloadSecurity"=dword:00000000
"HomepageLocation"="https://my.favourite.homepage.com"
```

**Comment by Darthagnon** [comment]
I based it off:
- https://learn.microsoft.com/en-us/deployedge/microsoft-edge-policies
- https://admx.help/?Category=EdgeChromium&Policy=Microsoft.Policies.Edge::PasswordMonitorAllowed_recommended

**Reply by Suncoast-Cyclist** [reply]
Fantastic references.  Thanks for sharing them.

**Reply by SegoCode** [reply]
this can be combined with a few host blocking https://gist.github.com/niutech/1f1c1518ce0eba7e8d429c812d39493d

**Comment by Darthagnon** [comment]
@TheBobPony I do wish this could have remained under issues, rather than closed without comment and dismissed to Github Discussions: where suggestions go to die™ 

I was trying to help and propose some more registry keys.

**Reply by TheBobPony** [reply]
Sorry if it came out as rude to you, I prefer suggestions like ideas to be under discussions. And well you know bugs and issues, to be under the issue tab. Hope you understand, thanks!

**Reply by SegoCode** [reply]
Well, the discussions have been focused on ideas without technical documentation, making their implementation and scope unclear and undefined. Once the scope is clear, it is technically documented in the issues as new functionality. From there, it can be tagged feature, invalid or approved and associated with its corresponding PR.

In this case, since @Darthagnon has technically documented the implementation and the scope is clear, it would be better to create an issue. This way, someone from the community can implement it and create the associated PR.

that's why this button exists:
![imagen](https://github.com/TheBobPony/MSEdgeTweaker/assets/35817798/178a2081-60fc-4e68-8250-18213bc8458c)

**Comment by Darthagnon** [comment]
See also https://github.com/TheBobPony/MSEdgeTweaker/issues/12 - people will suggest additional reg keys as Edgium grows in scope, and most will open new issues and not see "Discussions" because, though functionally identical to "Issues", they are a new anti-feature with poor visibility.
